pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -191,7 +190,7 @@ func (pl *DefaultPreemption) FindCandidates(ctx context.Context, state *framewor
 		return nil, err
 	}
 	if len(allNodes) == 0 {
-		return nil, core.ErrNoNodesAvailable
+		return nil, fmt.Errorf("no nodes available")
```

Comment:
@Huang-Wei  we should avoid returning errors and perhaps replace with status to avoid incorrectly reporting errors in the attempts metric. Perhaps we need a followup PR to make sure this is the case in this file?

Reply:
SGTM, so that we can filter out the abnormal errors.